ui/app/templates/components/generate-credentials-totp.hbs

```diff
@@ -0,0 +1,53 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<PageHeader as |p|>
+  <p.top>
+    <Hds::Breadcrumb>
+      <Hds::Breadcrumb::Item
+        @text={{@backendPath}}
+        @route="vault.cluster.secrets.backend"
+        @model={{@backendPath}}
+        data-test-link="role-list"
+      />
+      <Hds::Breadcrumb::Item @text="Codes" @route="vault.cluster.secrets.backend" @model={{@backendPath}} />
+      <Hds::Breadcrumb::Item @text={{@keyName}} @route="vault.cluster.secrets.backend.show" @model={{@keyName}} />
+      <Hds::Breadcrumb::Item @text={{this.title}} @current={{true}} />
+    </Hds::Breadcrumb>
+  </p.top>
+  <p.levelLeft>
+    <h1 data-test-title class="title is-3">
+      {{this.title}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<div
+  class="box is-fullwidth is-sideless is-paddingless is-marginless"
+  {{did-insert this.startTimer}}
+  {{will-destroy this.cancelTimer}}
```

Comment:
with ember concurrency we shouldn't need these. the task can be initiated in the `constructor()` for the component and are automatically canceled when the component is torn down.